www/pages/links.html.erb

```diff
@@ -0,0 +1,133 @@
+
+  <a name="articles"></a>
+  <h2 class="h2">Articles</h2>
+
+  <div class="row">
+    <div class="row__item">
+      <h3 class="h3">Web</h3>
+      <ul class="ul">
+        <li class="ul__li">
+          <a class="a" href="http://rubydev.ru/2011/10/interview_jeremy_evans_about_sequel_ru/">rubydev.ru: Interview with Jeremy Evans about Sequel and not only</a>
+          October 2011, Russian Translation
+        </li>
+        <li class="ul__li">
+          <a  class="a" href="http://on-ruby.blogspot.com/2009/01/sequel-interview-with-jeremy-evans.html">On Ruby: Sequel Interview with Jeremy Evans</a>
+          January 2009
+        </li>
+        <li class="ul__li">
+          <a  class="a" href="http://www.infoq.com/news/2008/11/sequel-ruby-db-toolkit">InfoQ: Sequel Interview with Jeremy Evans</a>
+          November 2008
+        </li>
+        <li class="ul__li">
+          <a  class="a" href="http://on-ruby.blogspot.com/2008/01/sequel-interview-with-sharon-rosner.html">On Ruby: Sequel Interview with Sharon Rosner</a>
+          January 2008
+        </li>
+        <li class="ul__li">
+          <a  class="a" href="http://www.oreillynet.com/ruby/blog/2007/07/ruby_project_spotlight_june_07.html">O"Reilly Ruby: Ruby Project Spotlight: Sequel</a>
+          July 2007
+        </li>
+      </ul>
+    </div>
+    <div class="row__item">
+      <h3 class="h3">Print</h3>
+      <ul class="ul">
+        <li class="ul__li">
+          <a  class="a" href="https://web.archive.org/web/20110209051845/http://www.impressjapan.jp/books/2579">Ruby! Vol. 1</a>
+          In japanese (thanks Shunichi Arai)
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <hr class="hr" />
+
+  <a name="presentations"></a>
+  <h2 class="h2">Presentations</h2>
+  <div class="row">
+    <div class="row__item">
+      <ul class="ul">
+        <li class="ul__li"><strong>RubyDay 2020</strong> <a class="a" href="https://speakerdeck.com/janko_m/sequel-when-activerecord-is-not-enough">Sequel: When Active Record is not enough</a></li>
+        <li class="ul__li"><strong>RubyC 2014</strong> <a class="a" href="http://code.jeremyevans.net/presentations/rubyc2014-2/index.html">Give-and-Go with PostgreSQL and Sequel</a> <a class="a" href="http://www.youtube.com/watch?v=toAcnwqlU1Q">Video</a></li>
+        <li class="ul__li"><strong>RubyC 2014</strong> <a class="a" href="http://code.jeremyevans.net/presentations/rubyc2014-1/index.html">Deep Dive Into Eager Loading Limited Associations</a> <a class="a" href="http://www.youtube.com/watch?v=KbF7RMk_2Qo">Video</a></li>
+        <li class="ul__li"><strong>RailsClub 2013</strong> <a class="a" href="http://code.jeremyevans.net/presentations/railsclub2013/index.html?trans=no">Give-and-Go with PostgreSQL and Sequel</a> <a class="a" href="http://code.jeremyevans.net/presentations/railsclub2013/index.html">In Russian</a> <a class="a a--dead" href="https://web.archive.org/web/20170915000000*/http://live.digicast.ru/ru/view/2116">Video</a></li>
```

Comment:
The site hosting the video is no more and I couldn't find another link for the video but I decided to link to the archived page anyways to reference there was a video at some point.